platform/iphone/app_delegate.mm

```diff
@@ -301,6 +304,7 @@ - (void)setControllerInputHandler:(GCController *)controller {
 				OSIPhone::get_singleton()->joy_axis(joy_id, JOY_ANALOG_R2, jx);
 			};
 		};
+		/*
```

Comment:
Should this code be removed instead of commented out?

And what about the following `#ifdef ADD_MICRO_GAMEPAD` which references being disabled by default due to not having API 9? I guess it's an outdated comment, but does this code work?

Reply:
Well, it's currently commented out at `4.0`, so I've decided to keep it. Will delete it after rebase.
As for `ADD_MICRO_GAMEPAD` Godot is already built for `SDK 10`, so it should be safe to use.

Reply:
Note that nothing defines `ADD_MICRO_GAMEPAD` currently, so this code might require testing to validate it before enabling it.

Reply:
Yeah, I've noticed that. While compilation seems to be working, I can't really test micro (?) gamepad as I don't have one.